pkg/manager/manager.go

```diff
@@ -70,6 +74,8 @@ func NewFraktiManager(
 	streamingServer streaming.Server,
 	privilegedRuntimeService runtime.RuntimeService,
 	privilegedImageService runtime.ImageService,
+	unikernelRuntimeService runtime.RuntimeService,
+	unikernelImageService runtime.ImageService,
```

Comment:
em...That looks better if we add more runtime. How can we decide which is unikernelRuntime or other extra runtimes and assign to corresponding field in `FraktiManager`?